sdk/schemaregistry/azure-data-schemaregistry/src/main/java/com/azure/data/schemaregistry/implementation/models/SchemasGetByIdHeaders.java

```diff
@@ -60,13 +60,19 @@ public final class SchemasGetByIdHeaders {
      * @param rawHeaders The raw HttpHeaders that will be used to create the property values.
      */
     public SchemasGetByIdHeaders(HttpHeaders rawHeaders) {
-        this.schemaVersion = Integer.parseInt(rawHeaders.getValue("Schema-Version"));
+        String schemaVersion = rawHeaders.getValue("Schema-Version");
+        if (schemaVersion != null) {
+            this.schemaVersion = Integer.parseInt(schemaVersion);
+        }
         this.schemaId = rawHeaders.getValue("Schema-Id");
         this.schemaGroupName = rawHeaders.getValue("Schema-Group-Name");
         this.schemaName = rawHeaders.getValue("Schema-Name");
         this.schemaIdLocation = rawHeaders.getValue("Schema-Id-Location");
         this.location = rawHeaders.getValue("Location");
-        this.contentType = rawHeaders.getValue("Content-Type");
+        String contentType = rawHeaders.getValue("Content-Type");
+        if (contentType != null) {
```

Comment:
same as above.

Reply:
This one wouldn't have thrown an NPE because it used to be a string, so we wouldn't try to manipulate it.  They changed the type in swagger to represent a class.